packages/code.rb

```diff
@@ -24,8 +24,8 @@ class Code < Package
   depends_on 'sommelier'
 
   case ARCH
-  when 'aarch64', 'armv7l'
-    @arch = 'arm64'
+  when 'armv7l', 'aarch64'
+    @arch = 'arm'
```

Comment:
This won't help.  There is only support for arm64.  Take a look at the code sometime.